vault/resource_azure_secret_backend.go

```diff
@@ -0,0 +1,183 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+func azureSecretBackendResource() *schema.Resource {
+	return &schema.Resource{
+		Create: azureSecretBackendCreate,
+		Read:   azureSecretBackendRead,
+		Update: azureSecretBackendCreate,
+		Delete: azureSecretBackendDelete,
+		Exists: azureSecretBackendExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"path": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Default:     "azure",
+				Description: "Path to mount the backend at.",
+				ValidateFunc: func(v interface{}, k string) (ws []string, errs []error) {
+					value := v.(string)
+					if strings.HasSuffix(value, "/") {
+						errs = append(errs, fmt.Errorf("path cannot end in '/'"))
+					}
+					return
+				},
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					return old+"/" == new || new+"/" == old
+				},
+			},
+			"description": {
```

Comment:
Question! I note that this isn't a field that's present on this API endpoint in Vault. Just curious about why it's added, and also why `ForceNew` is true. Doesn't that mean that if the description were to change, the object would be deleted and recreated remotely?

Reply:
All mounts can have a description added (https://www.vaultproject.io/api/system/mounts.html#description) but yes, probably shouldn't be ForceNew, let me fix that

Reply:
Fixed! 😄